proto/system_config.proto

```diff
@@ -183,11 +183,17 @@ message SwitchSetup {
   // Local port of second DAQ OpenFlow controller
   int32 lo_port_2 = 15;
 
-  // Local port for an alternate OpenFlow controller
+  // Port for an alternate stack setup, not controlled by daq.
   int32 alt_port = 16;
 
+  // Port for alternate setup varz server.
+  int32 varz_port = 17;
```

Comment:
Would `alt_varz_port` be better? I had thought this was to configure the prometheus port for faucet/gauge before I read the comment above...

Reply:
Good catch. I renamed this to alt_varz_port and the previous one to alt_of_port